Context/Users/Storage/Context/Users/User.ts

```diff
@@ -0,0 +1,25 @@
+import { cryptly } from "cryptly"
+import { isoly } from "isoly"
+import { userwidgets } from "@userwidgets/model"
+import { isly } from "isly"
+import { Permissions } from "./Permissions"
+
+export interface User extends Omit<userwidgets.User.Creatable, "password" | "permissions"> {
+	permissions: Permissions
+	password: cryptly.Password.Hash
+	created: isoly.DateTime
+	modified: isoly.DateTime
+}
+export namespace User {
+	export const type = isly.object<User>({
+		email: userwidgets.Email.type,
+		name: userwidgets.User.Name.type,
+		permissions: Permissions.type,
+		password: isly.fromIs("cryptly.Password.Hash", cryptly.Password.Hashed.is),
```

Comment:
Do we need this when we are omitting it?

Reply:
the property we omit is of the wrong type for this interface.  this is the correct type. question is if the omitting is required. I am keeping the omit for clarity.

Reply:
Yes, That's my bad, I had a look at it but forgot to remove the comment. Sorry.